test/utils/pickers/viewHandlers.ts

```diff
@@ -0,0 +1,58 @@
+import { fireTouchChangedEvent, userEvent, screen } from '@mui/monorepo/test/utils';
+import { getClockTouchEvent } from 'test/utils/pickers-utils';
+import { MuiPickersAdapter, TimeView } from '@mui/x-date-pickers/models';
+
+type TDate = any;
+
+interface ViewHandler<TView> {
+  setViewValue: (utils: MuiPickersAdapter<any>, viewValue: TDate, view?: TView) => void;
```

Comment:
I'm creating an object because I'd like to add other methods like `getViewValue` (or `assertViewValue` depending on the abstraction we want).